tests/foreman/cli/test_repository_set.py

```diff
@@ -147,3 +149,118 @@ def test_repositoryset_available_repositories(self):
             sum(int(repo['enabled'] == u'true') for repo in result.stdout),
             0
         )
+
+    def test_repositoryset_enable_by_name(self):
+        """@Test: Enable repo from reposet by names of reposet, org and product
+
+        @Feature: Repository-set
+
+        @Assert: Repository was enabled
+
+        """
+        org = make_org()
+        manifest = manifests.clone()
+        upload_file(manifest, remote_file=manifest)
+        result = Subscription.upload({
+            u'file': manifest,
+            u'organization-id': org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        result = RepositorySet.enable({
+            u'name': REPOSET['rhva6'],
+            u'organization': org['name'],
+            u'product': PRDS['rhel'],
+            u'releasever': '6Server',
+            u'basearch': 'x86_64',
+        })
+        self.assertEqual(result.return_code, 0)
+        result = RepositorySet.available_repositories({
+            u'name': REPOSET['rhva6'],
+            u'organization': org['name'],
+            u'product': PRDS['rhel'],
+        })
+        self.assertEqual(result.return_code, 0)
+        for repo in result.stdout:
+            if repo['release'] == '6Server' and repo['arch'] == 'x86_64':
+                enabled = repo['enabled']
```

Comment:
Personally, don't like this way of searching. Was thinking about something like this:
```python
enabled = filter(
    lambda repo: repo['arch'] == 'x86_64' and repo['release'] == '6Server',
    result.stdout
)[0]['enabled']
```
but not sure if lambdas are OK as they aren't used anywhere in our tests. Any suggestions?